104. Maximum Depth of Binary Tree.py

```diff
@@ -0,0 +1,22 @@
+# Definition for a binary tree node.
+# class TreeNode:
+#     def __init__(self, x):
+#         self.val = x
+#         self.left = None
+#         self.right = None
+
+class Solution:
+    def maxDepth(self, root):
+        """
+        :type root: TreeNode
+        :rtype: int
+        """
+        def findDepth(root):
+            if not root:
+                return 0
+            left = 1 + findDepth(root.left)
+            right = 1 + findDepth(root.right)
+            
+            return max(left,right)
```

Comment:
since you only use `left`, `right` once, can just do
`return 1 + max(findDepth(root.left), findDepth(root.right)`